src/OSPSuite.TeXReporting/TeX/CompilerConfiguration.cs

```
@@ -62,7 +53,11 @@ public static string MikTEXPortablePath
                if (!string.IsNullOrEmpty(path))
                   return path;
 
-               return Environment.GetEnvironmentVariable(MIK_TEX_INSTALL_DIR);
+               path = Environment.GetEnvironmentVariable(MIK_TEX_INSTALL_DIR);
+               if (!string.IsNullOrEmpty(path))
+                  return path;
+
+               throw new MikTexInstallationException();
```

need to throw here if not found!